src/main/kotlin/com/bugsnag/android/gradle/internal/ReactNativeUtils.kt

```diff
@@ -0,0 +1,45 @@
+package com.bugsnag.android.gradle.internal
+
+import com.android.build.gradle.api.ApkVariantOutput
+import org.gradle.api.Action
+import org.gradle.api.Describable
+import org.gradle.api.Project
+import org.gradle.api.Task
+import java.io.File
+
```

Comment:
nit: I feel like this might fit better in `BugsnagUploadJsSourceMapTask` as then all the sourcemap code remains in one place. I'm in two minds so will leave the decision up to you